infrastructure/team-dev/aws/au/cluster-config/nginx.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: routing
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-html
+  namespace: routing
+data:
+  index.html: hello from aws-au!
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: routing
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      volumes:
+      - name: nginx-html
+        configMap:
+          name: nginx-html
+      containers:
+      - name: nginx
+        image: nginx:1.25.4
+        ports:
+        - containerPort: 80
+        volumeMounts:
+        - mountPath: /usr/share/nginx/html
+          name: nginx-html
+      topologySpreadConstraints:
```

Comment:
@MattSurabian / @DanHartman (can't ping subbu yet because he hasn't joined the org), this is the current best practice for spreading workloads across nodes based on labels without causing scheduling headaches (anti-affinity can cause pods not to be scheduled once there is one on every label you're trying to spread over).

Reply:
@subbu-wescaleout testing testing, is this thing on? Seems like it should work, this is a public repo and that account exists but it isn't highlighting.